components/fms-forwarder/src/mqtt_connection.rs

```diff
@@ -139,6 +139,7 @@ pub fn add_command_line_args(command: Command) -> Command {
 
 /// A connection to an MQTT endpoint.
 ///
+#[allow(dead_code)]
```

Comment:
Being that this is part of the fms-forwarder I am not sure what is this struct doing however the reason why I had to add this was because otherwise the compiler would throw an error and therefore not pass the linting tests.